lms/djangoapps/certificates/api.py

```diff
@@ -320,7 +320,7 @@ def cert_generation_enabled(course_key):
     """
     return (
         CertificateGenerationConfiguration.current().enabled and
```

Comment:
What is this? Why does the other check need a course_key but this does not?

Reply:
I believe this checks whether self-generation is on at a site level.